HPXMLtoOpenStudio/resources/schedule_files/battery.csv

```diff
@@ -0,0 +1,8761 @@
+battery_charging,battery_discharging
+0,0
```

Comment:
It would be nice to have some fractions in here to verify that works. It looks like this file is all zeroes and ones.